tests/sentry/db/models/fields/slug/test_sentry_slug_field.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from sentry.db.models.fields.slug import NoNumericValidator, SentrySlugField
+from sentry.testutils.cases import TestCase
+
+
+class TestDefaultValidators(TestCase):
+    def setUp(self) -> None:
+        self.field = SentrySlugField()
+
+    def test_no_numeric_validator_added(self):
+        validators = [validator.__class__ for validator in self.field.validators]
+
+        # Ensure that NoNumericValidator is in the validators list
+        self.assertIn(NoNumericValidator, validators)
```

Comment:
this tests the _implementation_ and not the _behaviour_ so it's not a useful test -- I would delete this since the test below covers the actual _behaviour_

Reply:
Totally fair, updated!